src/lib/db/clients/sqlite.js

```diff
@@ -107,6 +109,48 @@ export function query(conn, queryText) {
   };
 }
 
+export async function updateValues(conn, updates) {
+  const updateCommands = updates.map(update => {
+    return {
+      query: `UPDATE ${update.table} SET ${update.column} = ? WHERE ${update.pkColumn} = ?`,
+      params: [update.value, update.primaryKey]
+    }
+  })
+
+  const commands = [{ query: 'BEGIN'}, ...updateCommands, {query: 'COMMIT'}];
```

Comment:
Does this makes sense ?
As we use `begin`, then the `update queries` and instantly `commit` the `rollback` on the  `catch` part will have no effect or not ?

I added the `commit` to the end in #277

Reply:
So I guess we should commit a the end, after the select queries, so it all happens in one go. I'll make that change.